src/assets/eye.-open.ts

```diff
@@ -0,0 +1,20 @@
+import { svg } from 'lit';
+
+export default svg`
+  <svg 
+    height="16" 
+    viewBox="0 0 22 16" 
+    width="22" 
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g fill="none" fill-rule="evenodd">
+      <g stroke="#333" stroke-width="1.5" transform="translate(1 2)">
+        <path 
+          d="m0 6c4.44444444-4 7.77777778-6 10-6 2.2222222 0 5.5555556 2 10 6-4.4444444 4-7.7777778 6-10 6-2.22222222 0-5.55555556-2-10-6z" 
+          stroke-linejoin="round"
+        />
+        <circle cx="10" cy="6" r="5"/>
+      </g>
+      <circle cx="11" cy="8" fill="#333" r="2"/>
+    </g>
+  </svg>`;
```

Comment:
Typo: I think the filename has an unwanted extra `.` in it (`src/assets/eye.-open.ts`)